src/main/webapp/room.js

```diff
@@ -0,0 +1,19 @@
+let players = [];
+
+const createUser = (ev)=>{
+    ev.preventDefault();  //to stop the form submitting
+    let user = {
+        username: document.getElementById('new-player-id').value,
+        get userID() {
+            // Base 36 (numbers + letters), and grab the first 9 characters after decimal.
+            return '_' + Math.random().toString(36).substr(2, 9);
+        }
+    }
+    players.push(user);
+
+    //saving to cookies
+    document.cookie = 'userID=' + user.userID + '; expires=' + new Date(2025, 0, 1).toUTCString();
+}
+document.addEventListener('DOMContentLoaded', ()=>{
+    document.getElementById('new-player-btn').addEventListener('click', createUser);
```

Comment:
the way this is currently setup, one user can create multiple "players" by clicking the button a bunch of times. I'm assuming you will handle this is future PRs right?

Reply:
@alekseychaikovsky Yes I will handle it in the future